arangod/MMFiles/MMFilesRestReplicationHandler.cpp

```diff
@@ -915,7 +929,13 @@ void MMFilesRestReplicationHandler::handleCommandDump() {
   bool includeSystem = _request->parsedValue("includeSystem", true);
   bool withTicks = _request->parsedValue("ticks", true);
 
-  ExecContextSuperuserScope escope(ExecContext::current().isAdminUser());
```

Comment:
@graetzer @jsteemann what is protected by this? The _users collection? The _wal? Why is a new scope needed?